.github/workflows/deploy-2-start.yml

```diff
@@ -89,6 +89,26 @@ jobs:
             spack.yaml \
             ${{ secrets.USER }}@${{ secrets.HOST_DATA }}:${{ vars.SPACK_YAML_LOCATION }}/${{ inputs.model }}.spack.yaml
 
+      - name: Fetch git sources for packages to build
+        # Workaround for ACCESS-NRI/spack#2.
+        # TODO: Remove once underlying spack issue is fixed
+        run: |
+          ssh ${{ secrets.USER }}@${{ secrets.HOST }} -i ${{ steps.ssh.outputs.private-key-path }} /bin/bash <<'EOT'
+
+          if [ ! -d "${{ steps.path.outputs.root }}/git_repos" ]; then
+            echo "::warning::Unable to find git_repos folder under ${{ steps.path.outputs.root }} - spack will attempt fetch instead"
+            exit 0
+          fi
+
+          cd "${{ steps.path.outputs.root }}/git_repos" || exit 1
+          # ls might not be space-separated all the time, xargs space-separates
+          for package_repo in $(ls | xargs); do
```

Comment:
I think 
```bash
for package_repo in *; do
```
should work too.

Reply:
Indeed it does! Will update it

Reply:
Done in https://github.com/ACCESS-NRI/build-cd/pull/189/commits/bd10d5511dd208955cff0d9a5ad12557c5e5cb44